api/v1/authentication_types.go

```diff
@@ -60,28 +65,31 @@ type AuthenticationOIDC struct {
 	// +required
 	UsernameClaim string `json:"usernameClaim"`
 
-	// RequestedScopes is a list of scopes to request from the OIDC provider. If not provided, all the available scopes
-	// are requested.
+	// RequestedScopes is a list of scopes to request from the OIDC provider. If not provided, the following scopes are
+	// requested: ["openid", "email", "profile", "groups", "offline_access"].
```

Comment:
These aren't the user-facing values, right?

We should change them to match what the user would put in:
- OpenID
- Email
- Profile
- Groups
- OfflineAccess

Reply:
This was a pre-existing field. To be honest I do not think we need it anymore at all and I have no plans of using it in the instructions for the new create-user-login.md page. I believe we could deprecate it, but when I made the design, I did not yet come to that realization.

That being said, the way I think it works and used to work is that the list provided would be passed into Kibana without modification. I realize that this is a violation of the standards for CRDs, where enum values have their first char capitalized.

The reason why I think it can be deprecated is because the manager uses a fixed set of scopes as well and those are limited to a subset of the current defaults. If this would not suffice, then people would not be able to log into the manager.

cc: @Brian-McM What are your thoughts on this?

Reply:
> This was a pre-existing field.

D'oh, right. Missed that! 

> list provided would be passed into Kibana without modification

Dang.

> The reason why I think it can be deprecated is because the manager uses a fixed set of scopes

Hm, does that effectively mean that the current code doesn't actually support changing this?

Reply:
This CRD was created for the Kibana OIDC feature in - I believe - v3.2. I think in that design Brian was very thorough in keeping all the options open for having a very flexible spec. While the code still uses this field to override the Kibana settings, it is not used for the manager settings. 

The manager just has a startup shell script that hard-codes [email profile groups]. 

If we think that we will need these requested scopes for the manager, we need to think about how to implement this. This would be a new feature that we did not have before. All that we really want from the identity provider are groups and a username and I believe with the current scopes we should be able to satisfy this.

I suppose we should take this back and discuss how to move forward with this field before making changes.

cc: @matthewdupre